client/app/visualizations/details/DetailsRenderer.jsx

```diff
@@ -0,0 +1,37 @@
+import React, { useState } from 'react';
+import map from 'lodash/map';
+import Pagination from 'antd/lib/pagination';
+
+import './details.less';
+
+export default function DetailsRenderer({ data }) {
+  const [page, setPage] = useState(0);
+
+  if (!data || !data.rows) {
+    return null;
+  }
+
+  // We use columsn to maintain order of columns in the view.
+  const columns = data.columns.map(column => column.name);
+  const row = data.rows[page];
+
+  return (
+    <div>
+      <table className="table table-bordered details-viz">
```

Comment:
@arikfr I assume you're not using Ant Table cause it has no "vertical header" feature.
I think I can implement this feature with CSS only - [`write-mode: lr;`](https://caniuse.com/#feat=css-writing-mode). Lmk if you see value in this attempt.

Reply:
Well, it was this and the fact it felt too much for this use case. But if we can make it work with just CSS, it can go along with my plan of merging this into the table visualization later on.

Not sure it's worth the effort at the moment though.

Reply:
Then let's leave it be. I never used this css method before and it might have unforeseen implications. The best way would be to PR Ant with `<Table orientation="vertical" />`.